frontend/talentsearch/src/js/components/reviewMyApplication/ReviewMyApplicationPage.tsx

```diff
@@ -0,0 +1,266 @@
+            editUrl: `${directIntakePaths.workLocation(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          workPreferences: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.workPreferences(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          employmentEquity: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.diversityEquityInclusion(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          roleSalary: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.roleSalary(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          skillsExperience: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.skillsAndExperiences(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+            override: (
+              <>
+                {missingSkills && (
+                  <div data-h2-margin="base(x1, 0)">
+                    <MissingSkills
+                      addedSkills={
+                        hasExperiences
+                          ? flattenExperienceSkills(experiences)
+                          : []
+                      }
+                      requiredSkills={missingSkills.requiredSkills}
+                      optionalSkills={missingSkills.optionalSkills}
+                    />
+                  </div>
+                )}
+                {!hasExperiences ? (
+                  <div
+                    data-h2-radius="base(s)"
+                    data-h2-background-color="base(light.dt-gray)"
+                    data-h2-padding="base(x1)"
+                  >
+                    <p data-h2-font-style="base(italic)">
+                      {intl.formatMessage({
+                        defaultMessage:
+                          "There are no experiences on your profile yet. You can add some using the preceding buttons.",
+                        description:
+                          "Message to user when no experiences have been attached to profile.",
+                      })}
+                    </p>
+                  </div>
+                ) : (
+                  <ExperienceSection experiences={experiences} />
+                )}
+                <div
+                  data-h2-display="base(flex)"
+                  data-h2-padding="base(x2, 0, 0, 0)"
+                  style={{ gap: "1rem" }}
+                >
+                  <Link
+                    href={directIntakePaths.signAndSubmit(applicant.id)}
+                    color="cta"
+                    mode="solid"
+                    type="button"
+                    disabled={!isApplicationComplete ?? false}
+                  >
+                    {intl.formatMessage({
+                      defaultMessage: "Continue to step 2",
```

Comment:
Missing the little arrow icon 
![image](https://user-images.githubusercontent.com/8978655/187977271-2b125f02-aa0a-4c01-9c26-a5d98b7baf2d.png)

Reply:
Fixed by [Improve styling to match designs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3796/commits/4c06217b89c30bf6548078c889c46d6de10ca560)